app/gateway/database/postgres/postgres.go

```diff
@@ -26,24 +30,20 @@ func Migrate(db string, filesPath string) error {
 		return errors.Wrap(err, "could not start migration")
 	}
 
-	if err := migration.Up(); err != nil && err != migrate.ErrNoChange && err != migrate.ErrInvalidVersion {
+	if err := migration.Up(); err != nil && err != migrate.ErrNoChange {
```

Comment:
Igor nesse caso não seria mais interessante usar a comparação do tipo do erro? (errors.Is()) Porque dessa forma, você compara o tipo do erro, e não a string informada em si.

Reply:
Resolvido!